test/tools/cluster.sh

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env bash
+
+if ! command -v mlaunch &> /dev/null
```

Comment:
@nbbeeken I snuck this in last minute is this cross platform?

Reply:
Yea this is fine! Actually this script also needs mongod/s to be in the user's path as well if you want to check for that also.